libs/hooks/open-telemetry/project.json

```diff
@@ -81,7 +81,9 @@
       ],
       "options": {
         "jestConfig": "libs/hooks/open-telemetry/jest.config.ts",
-        "passWithNoTests": true
+        "passWithNoTests": true,
+        "codeCoverage": true,
+        "coverageDirectory": "coverage/libs/hooks/open-telemetry"
```

Comment:
I wonder if we should add this to all projects, and the template.

Reply:
Yes, probably but I didn't want to pollute this PR.

Reply:
A follow up issue has been created.